gammapy/image/maps.py

```diff
@@ -34,14 +34,14 @@
 # Actually this class is generic enough to be rather in astropy and not in
 # gammapy and should maybe be moved to there...
 
-class SkyMap(object):
+class SkyImage(object):
     """
-    Base class to represent sky maps.
+    Base class to represent images.
```

Comment:
To me, "base class" always sounds like it's an abstract base class. But this is a normal class that can / should be instantiated. So I suggest to instead of
```
Base class to represent images.
```
put
```
Sky image.
```